client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -71,6 +78,24 @@ const ProductLightbox: React.FC< Props > = ( {
 		[ onChangeProduct, dispatch, siteId ]
 	);
 
+	const onChangeSocialProductOption = useCallback(
+		( productSlug ) => {
+			onChangeProduct( slugToSelectorProduct( productSlug ) );
+
+			// Tracking when variant selected inside the lightbox
+			dispatch(
+				recordTracksEvent( 'calypso_product_lightbox_variant_select', {
+					site_id: siteId,
+					product_slug: productSlug,
+				} )
+			);
+		},
+		[ onChangeProduct, dispatch, siteId ]
+	);
```

Comment:
Is it necessary to add this function separately for Social products? It seems identical to the `onChangeOption` function that's declared right above it.

Reply:
The typecheck `: string` got removed, is that intentional?